docs/ledger.md

```diff
@@ -136,3 +136,113 @@ Example:
 ```sh
 signatory-cli ledger set-high-watermark -d 3944f7a0 0
 ```
+
+## Vault Setup Hints
+- You CANNOT import a key to a ledger device
+- Making sure that the ledger device is able to work with your linux system is enabled through udev rules. Some examples can be found [here](https://github.com/LedgerHQ/udev-rules)
+  - I suggest appending the following to the section for the device you are using:
+  `, ACTION=="add", GROUP="{your_user_group}"`
+  - The add action is for when you plug in the device and making sure the device has the permissions group you belong to will make sure you don't need to run any commands as a root user
+
+## Key Management
```

Comment:
the entirety of this addition is repeating the information that is already on the page!

Reply:
what new information is being added by this PR on ledger.md?

Reply:
I have removed the information added to the ledger page as it is largely redundant, outdated, or hard to read.